sktime/forecasting/trend/tests/test_trend.py

```diff
@@ -144,3 +145,22 @@ def test_trendforecaster_with_datetimeindex():
 
     f = TrendForecaster()
     f.fit(df)
+
+
+def test_predict_var():
+    """Unit test for predict_interval method in PolynomialTrendForecaster.
+    We only need to test _predict_var() since the residuals are assumed normal.
+    The test:
+    Create series IID N(0,1), do a quadratic fit, and confirm Var(residuals) ~ 1
+    """
+    N = 500
+    data = np.random.normal(0, 1, N)
+    date_index = pd.date_range(start="1970-01-01", periods=N, freq="D")
+
+    df = pd.DataFrame(data, index=date_index, columns=["x"])
+
+    forecaster = PolynomialTrendForecaster(degree=2, prediction_intervals=True)
+    forecaster.fit(df)
+    fh = ForecastingHorizon([1, 2, 3, 4, 5, 6], is_relative=True)
+    a = forecaster.predict_var(fh=fh)
+    np.testing.assert_allclose(a.iloc[0, 0], 1.0, rtol=0.5)
```

Comment:
Can you kindly explain what does this line test? I'm not sure what is `1.0` here.

Reply:
it is the expected variance.

May I suggest a change to make this test "sharper"?

Let's generate a sample where we know the variance is *exactly* one!

How we could do this:
* set `data[42]` to 1 and `data[43]` to -1 to avoid sporadic numerical explosion
* do an in-sample fit of `PolynomialTrendForecaster` with prediction intervals off, to `data`, and compute residuals, with `predict_residuals`
* compute `s = np.std(data - residuals)`
* replace `data` by `data / s`

Reply:
Thanks @yarnabrina and @fkiraly. I implemented those suggestions. (fyi @fkiraly that should be `s = np.std(residuals)`)